andela/travel_tool_front
# #161376681 Change date header name

#### What does this PR do?
This PR ensures that start date is changed to Date created in the table

#### Description of Task to be completed?
- change date header name from `start date` to `Date Created`

#### How should this be manually tested?
- Clone the repo - git clone https://github.com/andela/travel_tool_front.git
- Change into the project directory
- Checkout to the branch with git fetch -a then git checkout ft-delete-travel-checklist-item-fix-161044598
- Pull [develop](https://github.com/andela/travel_tool_back) on the backend
- Run the server with make start
- Navigate to Requests page and create a `New Request`
- you should be able to see the table with the name changed

#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?
[#161376681](https://www.pivotaltracker.com/story/show/161376681)

#### Screenshots (if appropriate)
<img width="1411" alt="screenshot 2018-10-29 at 11 38 36 am" src="https://user-images.githubusercontent.com/26988297/47644901-b63b6a00-db6f-11e8-847d-2b428b970f50.png">


#### Questions:
